ceno_zkvm/src/instructions/riscv/slti.rs

```diff
@@ -105,8 +105,8 @@ impl<E: ExtensionField, I: RIVInstruction> Instruction<E> for SetLessThanImmInst
             .rs1_read
             .assign_value(instance, Value::new_unchecked(rs1));
 
-        let imm = step.insn().imm_internal();
-        let imm_field = InsnRecord::imm_internal_field::<E::BaseField>(&step.insn());
+        let imm = step.insn().immediate();
```

Comment:
I think we can replace it with `InsnRecord::process_imm` here and as well as in all the code above, to make it more unified.`InsnRecord::process_imm` seems like a place for special cases, similar to original `imm_internal()`. By unifying calls to `process_imm`, circuit writers won’t have to decide whether to use `step.insn().immediate()` or `InsnRecord::process_imm()`.

Reply:
Right! I will put it back that way.